### PR TITLE
Issue2182 pid

### LINKED ISSUE #2182: Revise CDL.Continuous.PID and PIDReset
This issue is to revise `CDL.Continuous.PID` and `PIDReset` to
- add a scaling factor for the control input,
- remove the set point weight `wp` and `wd` which have not been used (removing them also simplifies adding the scaling factor)
- update the documentation to avoid having to reference the MSL.

### PR DESCRIPTION
**Pull request by mwetter**
This closes #2182

### CODE REVIEW

File: Buildings/Controls/OBC/CDL/Continuous/PID.mo [hunk 1]
```diff
@@ -1,49 +1,69 @@
 within Buildings.Controls.OBC.CDL.Continuous;
 block PID
-  "P, PI, PD, and PID controller with limited output, anti-windup compensation and setpoint weighting"
+  "P, PI, PD, and PID controller"
```
Comment (JayHuLBL) [inline Buildings/Controls/OBC/CDL/Continuous/PID.mo:3]: I think the controller still have limited output and anti-windup compensation.

File: Buildings/Examples/DualFanDualDuct/Controls/RoomMixingBox.mo [hunk 2]
```diff
@@ -34,7 +34,8 @@ block RoomMixingBox "Controller for room mixing box"
     controllerType=Buildings.Controls.OBC.CDL.Types.SimpleController.PI,
     Td=60,
     k=0.1,
-    Ti=120) "Controller for mass flow rate"
+    Ti=120,
+    r=1/m_flow_min) "Controller for mass flow rate"
```
Comment (JayHuLBL) [inline Buildings/Examples/DualFanDualDuct/Controls/RoomMixingBox.mo:38]: should it be `r=m_flow_min`?

File: Buildings/Controls/OBC/ASHRAE/G36_PR1/AHUs/MultiZone/VAV/SetPoints/ExhaustDamper.mo [hunk 3]
```diff
@@ -40,8 +40,10 @@ block ExhaustDamper
     annotation (Placement(transformation(extent={{-30,50},{-10,70}})));
   Buildings.Controls.OBC.CDL.Continuous.PID conP(
     final controllerType=Buildings.Controls.OBC.CDL.Types.SimpleController.P,
-    final k=k) "Building static pressure controller"
-    annotation (Placement(transformation(extent={{40,50},{60,70}})));
+    final k=k,
+    r=dpBuiSet)
```
Comment (JayHuLBL) [inline Buildings/Controls/OBC/ASHRAE/G36_PR1/AHUs/MultiZone/VAV/SetPoints/ExhaustDamper.mo:44]: finalize `r` here and in other below classes:
```
  final r=dpBuiSet
```

**Review by JayHuLBL — COMMENTED** [review]
@mwetter I added inline comments.

File: Buildings/Controls/OBC/CDL/Continuous/PID.mo (hunk 1 above)
Comment (mwetter) [reply Buildings/Controls/OBC/CDL/Continuous/PID.mo:3]: Correct, but I removed it from the short description as both PID controllers have these features. The short descriptions are

* PID   -  P, PI, PD, and PID controller
* PIDWithReset  -  P, PI, PD, and PID controller with output reset

File: Buildings/Examples/DualFanDualDuct/Controls/RoomMixingBox.mo (hunk 2 above)
Comment (mwetter) [reply Buildings/Examples/DualFanDualDuct/Controls/RoomMixingBox.mo:38]: Correct, will be fixed.

File: Buildings/Controls/OBC/ASHRAE/G36_PR1/AHUs/MultiZone/VAV/SetPoints/ExhaustDamper.mo (hunk 3 above)
Comment (mwetter) [reply Buildings/Controls/OBC/ASHRAE/G36_PR1/AHUs/MultiZone/VAV/SetPoints/ExhaustDamper.mo:44]: Thanks, fixed.

**Pull request merged**